packages/app/src/components/transactions/index.tsx

```diff
@@ -302,5 +305,5 @@ const DateDisplay = styled.div`
 `;
 const HourDisplay = styled.div`
   font-size: 12px;
-  color: ${Colors.Gray["600"]};
+  color: gray;
```

Comment:
we can use the `theme` colors with `normalizeColor` helper in styled-components:
```
const HourDisplay = styled.div`
  ${({ theme }) => css`
    font-size: 12px;
    color: ${normalizeColor("color-from-theme", theme)};
  `}
`;
```